### DISCUSSION TITLE
How to use ENUM (all string)?

### DISCUSSION THREAD

**Discussion by axelcasauran** [serenity-is/Serenity#6278] [Help]
Hi,

I am implementing a combo box with all the country name. I followed [https://serenity.is/docs/tutorials/movies/adding_a_movie_kind_field](https://serenity.is/docs/tutorials/movies/adding_a_movie_kind_field) but it seems like it only works for enum with integer (id). Is there a way to make it work with all string only? (no ID or integer)

Thanks...

**Comment by dfaruque** [comment]
https://serenity.is/demo/BasicSamples/SelectWithHardcodedValues
